base/operators/openshift-special-resource-operator/01-install.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: openshift-special-resource-operator
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+ name: special-resource-operators
+ namespace: openshift-special-resource-operator
+spec:
```

Comment:
Should this be` spec: {} ` ? When empty it should have something indicating that is the case.

Reply:
updated